src/components/PreviewConcept/Concept.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { Remarkable } from 'remarkable';
+import {
+  NotionDialogContent,
+  NotionHeaderWithoutExitButton,
+  NotionDialogLicenses,
+  NotionDialogText,
+} from '@ndla/notion';
+import { Concept as ConceptType } from '../SlateEditor/editorTypes';
+import { getSrcSets } from '../../util/imageEditorUtil';
+
+const StyledBody = styled.div`
+  margin: 0 ${spacing.normal} ${spacing.small};
+  .c-tabs {
+    margin-left: 0;
+    margin-right: 0;
+  }
+  .react-tabs__tab-panel--selected {
+    animation-name: fadeInLeft;
+    animation-duration: 500ms;
+  }
+`;
+
+interface Props {
+  concept: ConceptType;
+}
+
+const Concept: FC<Props & tType> = ({ concept }) => {
+  const markdown = new Remarkable({ breaks: true });
+
+  const renderMarkdown = (text: string) => {
+    const rendered = markdown.render(text);
+    return (
+      <>
+        <span dangerouslySetInnerHTML={{ __html: rendered }} />
+      </>
```

Comment:
Denne trenger kanskje ikke wrappes i en fragment? da er det vel kanskje lite fint å bare legge den inn på linje 90 uten dette ekstra steget? 

```suggestion
        <span dangerouslySetInnerHTML={{ __html: rendered }} />
```

Reply:
Ja for det går kanskje greit å bare rendre markdown globalt i komponenten?
```
const Concept: FC<Props & tType> = ({ concept }) => {
  const markdown = new Remarkable({ breaks: true });
  const renderedContent = markdown.render(concept.content);

...

<NotionDialogText>
   <span dangerouslySetInnerHTML={{ __html: renderedContent }} />
</NotionDialogText>
```

Reply:
Fullt mulig det er greit å gjøre det slik.

Reply:
Samtidig treng du ikkje renderedContent nokon plass så det gir ikkje heilt meining å ta vare på det som en const. Då er det bedre å ha det som det er direkte i return-koden.